src/databricks/labs/ucx/source_code/known.py

```diff
@@ -56,13 +56,22 @@ def __init__(self):
             self._module_problems[name] = []
 
     @staticmethod
-    def _get_known():
-        module = __name__
-        if __name__ == "__main__":  # code path for UCX developers invoking `make known`
-            module = "databricks.labs.ucx.source_code.known"
-        # load known.json from package data, because we may want to use zipapp packaging
-        data = pkgutil.get_data(module, "known.json")
-        return json.loads(data)
+    def _get_known() -> dict[str, dict[str, list[dict[str, str]]]]:
+        """Loads the set of known distributions (and false positives), included with this module.
+
+        Returns:
+            A dictionary (keyed by distribution name) containing the known problems for each module.
+        Raises:
+            FileNotFoundError: If the builtin set of known distribution is missing.
+        """
+        this_module = __package__
```

Comment:
1. does it work for `make known`? 
2. does it work when the script is debugged directly?

Reply:
Yes, and yes.